Author: Cesare Falco <c.falco@ubuntu.com>
Description: Add new option -listmidi to Mame and Mess man pages
Forwarded: yes

Index: mame-0.148/src/osd/sdl/man/mame.6
===================================================================
--- mame-0.148.orig/src/osd/sdl/man/mame.6	2012-12-06 05:39:12.000000000 +0100
+++ mame-0.148/src/osd/sdl/man/mame.6	2013-01-24 21:56:13.000000000 +0100
@@ -13,7 +13,7 @@
 .\" and updated by Andrew Burton <burtona@gol.com>, July 2003
 .\"
 .\"
-.TH MAME 6 2012-12-01 0.147u3 "MAME \- The Multiple Arcade Machine Emulator"
+.TH MAME 6 2013-01-15 0.148 "MAME \- The Multiple Arcade Machine Emulator"
 .\"
 .\"
 .\" NAME chapter
@@ -163,6 +163,9 @@
 .TP
 .B \-verifysoftlist, \-vlist
 Verify software list by name.
+.TP
+.B \-listmidi, \-mlist
+List available MIDI I/O devices.
 .\"
 .\" *******************************************************
 .SS Configuration options
@@ -926,7 +929,7 @@
 Specifies the key used to toggle between full and partial UI mode.
 .TP
 .B \-keymap
-Enable keymap for non\-QWERTY keyboards. Used in conjuction with
+Enable keymap for non\-QWERTY keyboards. Used in conjunction with
 \fB\-keymap_file\fR.
 .TP
 .B \-keymap_file \fIkeymap_file\fR
Index: mame-0.148/src/osd/sdl/man/mess.6
===================================================================
--- mame-0.148.orig/src/osd/sdl/man/mess.6	2012-12-20 01:56:47.000000000 +0100
+++ mame-0.148/src/osd/sdl/man/mess.6	2013-01-24 21:54:48.000000000 +0100
@@ -16,7 +16,7 @@
 .\" http://www.mess.org/
 .\"
 .\"
-.TH mess 6 2012-12-01 0.147u3 "The Multiple Emulator Super System (MESS)"
+.TH mess 6 2013-01-15 0.148 "The Multiple Emulator Super System (MESS)"
 .\"
 .\"
 .\" NAME chapter
@@ -162,6 +162,9 @@
 .TP
 .B \-verifysoftlist, \-vlist
 Verify software list by name.
+.TP
+.B \-listmidi, \-mlist
+List available MIDI I/O devices.
 .\"
 .\" *******************************************************
 .SS Configuration options
